rviz_ogre_vendor/pragma-patch.diff

```diff
@@ -1845,4 +1845,20 @@ diff --git a/OgreMain/include/OgreTimer.h b/OgreMain/include/OgreTimer.h
 +#pragma warning(pop)
 +#endif
  #endif
-
+diff --git a/CMake/Utils/PrecompiledHeader.cmake b/CMake/Utils/PrecompiledHeader.cmake
+index bfb0059..62b4f6d 100644
+--- a/CMake/Utils/PrecompiledHeader.cmake
++++ b/CMake/Utils/PrecompiledHeader.cmake
+@@ -340,7 +340,11 @@ MACRO(ADD_NATIVE_PRECOMPILED_HEADER _targetName _input)
+     if(NOT OGRE_ENABLE_PRECOMPILED_HEADERS)
+         # do nothing
+     elseif(CMAKE_VERSION GREATER_EQUAL "3.16")
+-        target_precompile_headers(${_targetName} PRIVATE ${_input})
++        if(APPLE)
++            add_compile_options((-Xclang -include -Xclang ${_input}))
++        else()
++            target_precompile_headers(${_targetName} PRIVATE ${_input})
++        endif()

```

Comment:
Hi, can we bring this patch back? currently it's failing on rolling branch and applying this diff seems to fix it. here's the error:

```
Starting >>> rviz_ogre_vendor
--- stderr: rviz_ogre_vendor
error: Objective-C was disabled in PCH file but is currently enabled
1 error generated.
make[5]: *** [OgreMain/CMakeFiles/OgreMain.dir/src/OSX/macUtils.mm.o] Error 1
make[5]: *** Waiting for unfinished jobs....
make[4]: *** [OgreMain/CMakeFiles/OgreMain.dir/all] Error 2
make[3]: *** [all] Error 2
make[2]: *** [ogre_vendor-prefix/src/ogre_vendor-stamp/ogre_vendor-build] Error 2
make[1]: *** [CMakeFiles/ogre_vendor.dir/all] Error 2
make: *** [all] Error 2
---
Failed   <<< rviz_ogre_vendor [6.91s, exited with code 2]
```